src/snowflake/snowpark/modin/plugin/docstrings/series.py

```diff
@@ -2912,11 +2912,15 @@ def value_counts():
             frequencies of the unique values. Being different from native pandas,
             Snowpark pandas will return a Series with `decimal.Decimal` values.
         sort : bool, default True
-            Sort by frequencies when True. Preserve the order of the data when False.
+            Sort by the count when True. Preserve the order of the data when False.
```

Comment:
Let's also use frequencies here to stay consistent with pandas.

I think we want to add that "original data order is preserved when there are equal frequency" for sort = True part